tools/run_tests/artifacts/artifact_targets.py

```diff
@@ -95,7 +95,7 @@ def create_jobspec(name,
     return jobspec
 
 
-_MACOS_COMPAT_FLAG = '-mmacosx-version-min=10.7'
+_MACOS_COMPAT_FLAG = '-mmacosx-version-min=10.10'
```

Comment:
qq: is this really needed?

Reply:
This doesn't need to be part of this PR but I just make this consistent with the all other settings for OSX.